lightbulb/cooldowns.py

```diff
@@ -42,7 +42,20 @@
     from lightbulb import commands
     from lightbulb import context as context_
 
-    DynamicCallback = typing.Callable[[context_.Context], typing.Optional["Bucket"]]
+    BucketOrNoBucketOption = typing.Union["Bucket", "NoBucketOption"]
+    DynamicCallback = typing.Callable[[context_.Context], BucketOrNoBucketOption]
+
+
+class NoBucketOption(int, enums.Enum):
+    """
+    What to do when a dynamic cooldown callback should not return a new bucket.
+    This is useful in case you want to bypass cooldowns or reset them in certain conditions.
+    """
+
+    RESET = 0
+    """Resets all existing cooldowns"""
```

Comment:
Bad wording there, I mean `Resets all cooldowns for the given context`. It doesn't nuke all cooldowns, just the ones that fit the context the callback is invoked in.

Reply:
Dunno if that's a better way to phrase it. Doesn't feel 100% right to me either.

Reply:
It should be singular, shouldn't it? A context can only extract a hash—because there can only be a bucket at max for each context. Perhaps, "Resets the cooldown of a context if any."? I'm not the best at wording either 😅

Reply:
N/A anymore.